custom-plugins/grafana-worldmap-panel-msupply/src/worldmap.ts

```diff
@@ -215,7 +215,13 @@ export default class WorldMap {
 
   createPopup(circle, locationName, value) {
     const unit = value && value === 1 ? this.ctrl.panel.unitSingular : this.ctrl.panel.unitPlural;
-    const label = (locationName + ': ' + value + ' ' + (unit || '')).trim();
+    const useTemplate = !!this.ctrl.panel.labelTemplate;
+    const label = useTemplate
+      ? this.ctrl.panel.labelTemplate
+          .replace('${name}', locationName)
+          .replace('${value}', value)
+          .replace('${unit}', unit)
+      : (locationName + ': ' + value + ' ' + (unit || '')).trim();
```

Comment:
Just wondering, why not just:
```
const label = !!this.ctrl.panel.labelTemplate;
      ? this.ctrl.panel.labelTemplate
          .replace('${name}', locationName)
          .replace('${value}', value)
          .replace('${unit}', unit)
      : (locationName + ': ' + value + ' ' + (unit || '')).trim();
```
Just a stylistic/readability preference, or is there a difference?

Reply:
> Just a stylistic/readability preference, or is there a difference?

no difference - I've picked up the habit (from Will I think) of adding in variables in cases like this - it makes the intention clearer. I was also trying, not very successfully, to make the code block smaller

Reply:
Cool -- I did it the other way on the regional map panel. ;)